### PR TITLE
Adding Partner Section to k8s.io

### PR DESCRIPTION
**Pull request by ankurshukla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
 
 All versions of the site that relate to past and future versions will be named after their Kubernetes release number. For example, [the old branch for the 1.1 docs is called `release-1.1`](https://github.com/kubernetes/kubernetes.github.io/tree/release-1.1).
 
+## Partners {#partners}
```
Comment (johndmulhausen) [inline README.md]: You don't need to add {#partners} 
:)

File: README.md [hunk 2]
```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
...
+
+For Kubernetes partners, once verified by the community manager, can get their logos added to the partner section on [community page](http://k8s.io/community). The partner solutions should have a landing page / guide / blog at their end that talks about the integration details with kubernetes.
```
Comment (johndmulhausen) [inline README.md]: Partners can get their logos added to the partner section on [community page](http://k8s.io/community) by following the below steps. Partners will need to have a URL that is specific to integrating with Kubernetes; this URL will be the destination when the logo is clicked.

File: README.md [hunk 3]
```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
...
+
+* The partner product logo should be a transparent png image centered in a 215x125 px frame. (look at the existing logos for reference)
+* The partner solution link should take directly to the kubernetes integration landing page/doc/blog on partner's own site.
```
Comment (johndmulhausen) [inline README.md]: The logo must link to a URL that is specific to integrating with Kubernetes, hosted on the partner's site.

File: README.md [hunk 4]
```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
...
+* The logo should be named *product-name*_logo.png and placed at `/images/community_logos` folder.
```
Comment (johndmulhausen) [inline README.md]: The logo should be named `*product-name*_logo.png` and placed in the  `/images/community_logos` folder.

File: README.md [hunk 5]
```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
...
+* The image reference (and hyperlink)should be added in`community.html` under `<div class="partner-logos" > ...</div>`.
```
Comment (johndmulhausen) [inline README.md]: The image reference (including the link to the partner URL) should be added in `community.html` under `<div class="partner-logos" > ...</div>`.

File: README.md [hunk 6]
```diff
@@ -131,6 +131,17 @@ The current version of the website is served out of the `master` branch.
...
+* Please do not change the order of the existing partner images. Append your `<a href="..."><img ...></a>`tag at the end of the list.
```
Comment (johndmulhausen) [inline README.md]: Please do not change the order of the existing partner images. Append your logo the end of the list.

**Comment by johndmulhausen** [comment]
A few comments here; also you need to solve the merge conflict. :(

File: README.md (hunk 1 above)
Comment (ankurshukla) [inline README.md]: I added this to give "partners" an anchortag! 
 k8s.io/editdocs/#partners

PS: not a markdown expert :)

Comment (johndmulhausen) [inline README.md]: All headings automatically get an anchortag, at least in our Markdown processor. :)

**Comment by johndmulhausen** [comment]
Still got a merge conflict!

**Comment by ankurshukla** [comment]
@johndmulhausen I am not able to find any conflicts? 
git status
git fetch origin
git merge origin master

checked all.

**Comment by ankurshukla** [comment]
@johndmulhausen any updates or comments on this?

**Comment by johndmulhausen** [comment]
Hi @ankurshukla -- sorry, I was on vacation. Yes, there are still merge conflicts. On this page you should be seeing links to GitHub help on how to resolve the issue.

**Pull request closed without merging**